test/jdk/javax/swing/JColorChooser/Test6827032.java

```diff
@@ -24,21 +24,24 @@
 /*
  * @test
  * @key headful
- * @bug 6827032 8197825
+ * @bug 6827032
```

Comment:
[JDK-8197825](https://bugs.openjdk.org/browse/JDK-8197825) was a test stabilisation fix, we don't add such bugs into the `@bug` tag.